gremlin-go/docker/generate-all.groovy

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
we've agreed that we're trying to establish a testing pattern here that we can extend to all GLVs, so i'll continue to ignore this duplication. just commenting here to continue to keep it forefront that we're working towards that goal and that test configuration copies is not going to be the approach taken.